src/coreclr/gc/handletable.cpp

```diff
@@ -24,6 +24,19 @@
 DWORD g_dwHandles = 0;
 #endif // ENABLE_PERF_COUNTERS || FEATURE_EVENT_TRACE
 
+#ifndef DACCESS_COMPILE
+
+#ifndef INT_MAX
+#define INT_MAX       2147483647
+#endif
```

Comment:
Under what conditions is this value not set?

Reply:
Linux. the previous CI build indicates that and I can repro that locally.

Reply:
Okay. Thanks.

Reply:
Include the right header instead?